webapi/src/main/scala/org/knora/webapi/messages/v2/responder/ontologymessages/OntologyMessagesV2.scala

```diff
@@ -80,34 +80,17 @@ case class CreateOntologyRequestV2(
 /**
  * Constructs instances of [[CreateOntologyRequestV2]] based on JSON-LD requests.
  */
-object CreateOntologyRequestV2 extends KnoraJsonLDRequestReaderV2[CreateOntologyRequestV2] {
+object CreateOntologyRequestV2 {
```

Comment:
can you elaborate on this removal on our next backend meeting? seems nice to get rid of this, but it makes me wonder where else we should do this, now?

Reply:
The `KnoraJsonLDRequestReaderV2` trais has tied the `fromJsonLD` factory methods to a very strict form with parameters which are not needed in most places and this is the only definition in that `trait`. These factory methods where always used directly and therefore the interface did not provide any value other than tying the signature to something unnecessary.

Bit by bit I will get rid if when tackling the other nine places where it currently used.